test/world_cup_test.rb

```diff
@@ -0,0 +1,105 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/team.rb'
+require './lib/player.rb'
+require './lib/world_cup.rb'
+
+
+class WorldCupTest < Minitest::Test
```

Comment:
Seems like you might be writing all of your tests up front. Consider switching your workflow to write a test, then make it pass, then write a test, then make it pass, etc.